From: Krzesimir Nowak <qdlacz@gmail.com>
Date: Sat, 9 Jan 2021 16:00:46 +0100
Subject: tests: Update the expected count in file test

We added another desktop file, so update the file count.

Bug: https://gitlab.gnome.org/GNOME/glib/-/merge_requests/1872
Origin: upstream, 2.66.5, commit:4963b48a8482e504c224ac828d31b74099999b56
---
 gio/tests/file.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gio/tests/file.c b/gio/tests/file.c
index c3877af..d876965 100644
--- a/gio/tests/file.c
+++ b/gio/tests/file.c
@@ -1041,7 +1041,7 @@ test_measure (void)
   if (size > 0)
     g_assert_cmpuint (num_bytes, ==, size);
   g_assert_cmpuint (num_dirs, ==, 6);
-  g_assert_cmpuint (num_files, ==, 31);
+  g_assert_cmpuint (num_files, ==, 32);
 
   g_object_unref (file);
   g_free (path);
@@ -1131,7 +1131,7 @@ test_measure_async (void)
   g_free (path);
 
   data->expected_dirs = 6;
-  data->expected_files = 31;
+  data->expected_files = 32;
 
   g_file_measure_disk_usage_async (file,
                                    G_FILE_MEASURE_APPARENT_SIZE,
